styles.css

```diff
@@ -0,0 +1,23 @@
+
```

Comment:
This looks awesome! In terms of naming conventions I would consider being as explicit as possible when naming ```#ids``` or ```.classes```. ```my-image``` and ```another-image``` could be confusing to another developer checking your code or even yourself when you come back to this in two weeks!